handlers/auth.go

```diff
@@ -89,6 +101,17 @@ type AuthConfig struct {
 	RoleCategories []*gaia.UserRoleCategory
 }
 
+func getPoliciesFromClaims(policyClaims interface{}) []string {
+	policies := []string{}
```

Comment:
In this case, I'd prefer a named return value for clarity. But I'll leave it up to you. :)